Protacon.RxMq.AzureServiceBusLegacy/Topic/AzureBusTopicPublisher.cs

```diff
@@ -109,10 +109,93 @@ public Task SendAsync<T>(T message) where T : new()
             var queueName = _settings.TopicNameBuilder(message.GetType());
 
             if (!_bindings.ContainsKey(queueName))
-                _bindings.Add(queueName, new Binding<T>(_factory, _namespaceManager, _settings, queueName
-                    , _logMessage, _logError));
+                return TryCreateBinding(queueName, typeof(T), message, 5, 60);
```

Comment:
Give magical numbers some names. I would also recommend using `TimeSpan` directly here instead of int for seconds.

Reply:
Added these to class fields, also changed to pass TimeSpan directly